website/src/docs/hotchocolate/defining-a-schema/extending-types.md

```diff
@@ -303,6 +276,46 @@ Simply replace the field on the existing type.
 </ExampleTabs.Schema>
 </ExampleTabs>
 
+## Extending by name
+
+If we can not reference a type, we can still extend it by specifying its name.
+
+<ExampleTabs>
+<ExampleTabs.Annotation>
+
+```csharp
+[ExtendObjectType("Foo")]
+public class FooExtensions
+{
+    // Omitted code for brevity
+}
+```
+
+</ExampleTabs.Annotation>
+<ExampleTabs.Code>
+
+```csharp
+public class FooTypeExtensions : ObjectTypeExtension
+{
+    protected override void Configure(IObjectTypeDescriptor descriptor)
+    {
+        descriptor.Name("Foo");
+
+        // Omitted code for brevity
+    }
+}
+```
+
+</ExampleTabs.Code>
+<ExampleTabs.Schema>
+
+⚠️ Schema-first does currently not support extending types by their name
+
+</ExampleTabs.Schema>
+</ExampleTabs>
+
+When extending root types, we can make use of the constants in `OperationTypeNames`. We can for example use `OperationTypeNames.Query` instead of writing "Query" everywhere.
```

Comment:
```suggestion
When extending root types, we can make use of the constants in `OperationTypeNames`. We can for example use `OperationTypeNames.Query` instead of writing `"Query"` everywhere.
```